nav2_system_tests/src/waypoint_follower/tester.py

```diff
@@ -33,7 +33,7 @@ class WaypointFollowerTest(Node):
     def __init__(self):
         super().__init__(node_name='nav2_waypoint_tester', namespace='')
         self.waypoints = None
-        self.action_client = ActionClient(self, FollowWaypoints, 'FollowWaypoints')
+        self.action_client = ActionClient(self, follow_waypoints, 'follow_waypoints')
```

Comment:
As well here, the name should be changed, but the type should be reverted to FollowWaypoints